table/sst_file_reader_test.cc

```diff
@@ -90,6 +90,7 @@ class SstFileReaderTest : public testing::Test {
     if (check_global_seqno) {
       auto properties = reader.GetTableProperties();
       ASSERT_TRUE(properties);
+      ASSERT_FALSE(properties->db_host_id.empty());
```

Comment:
I'm not a fan of "copying the code into the unit test" as an oracle, but it might be worth comparing against GetHostName on the Env here. (I'm not too concerned about the case of someone changing their host name during this test case running.) Up to you